test/functional/api/v2/stacks_controller_test.rb

```diff
@@ -4,22 +4,21 @@ class ForemanDeployments::Api::V2::StacksControllerTest < ActionController::Test
   class FakeTask < ForemanDeployments::Tasks::BaseDefinition
   end
 
-  describe 'importing stack' do
-    setup do
-      @definition = [
-        'FirstRun: !task:FakeTask',
-        '  parameters:',
-        '    host_id: 1'
-      ].join("\n")
-    end
+  setup do
+    @definition = [
+      'FirstRun: !task:FakeTask',
+      '  parameters:',
+      '    host_id: 1'
+    ].join("\n")
+    ForemanDeployments.registry.register_task(FakeTask)
```

Comment:
Messing with global configuration can hurt other tests, it's better to isolate the registry.
I did something similar [here]( https://github.com/theforeman/foreman/pull/2665/files#diff-314ba154d26046781f3becacc34d9ddbR39).